gsa/src/web/pages/scanconfigs/nvtpreferences.js

```diff
@@ -65,15 +65,17 @@ class NvtPreferenceDisplay extends React.Component {
   }
 }
 
-NvtPreferenceDisplay.propTypes = {
-  preference: PropTypes.shape({
-    nvt: PropTypes.shape({
-      oid: PropTypes.string.isRequired,
-      name: PropTypes.string.isRequired,
-    }).isRequired,
+export const NvtPreferencePropType = PropTypes.shape({
```

Comment:
I think this should be defined in utils/proptypes.js. It's rather special and not used anywhere else, but it would make looking up proptype information much easier if all the proptypes that we have are defined together (in a place where  we already have specific proptypes like error, capabilities, icon etc).

Reply:
I am not sure about that. They aren't very generic and only used at these two places.

Reply:
I thought about that as well. But if that was completely true, there would be no need to export them (even if it was "only" for testing or so, this would make it more than one place, where it's used). The proptype iconSize, e.g. is also not very generic, despite being used in several places.

Reply:
Not generic did mean it is has a very special use case and will not be used outside of these both components.

Reply:
I've thought about your comment about the proptypes again and I don't think we should add them to the proptypes module because they are really specific to the used components in this PR. The preferences do contain more data then displayed and used in these components here. So using them at possible other places would require to extend them. But in that case they would also require more data then needed here and the test cased have to extended to avoid proptype warnings.